bin/export-db-to-sqlite.sh

```diff
@@ -173,7 +173,6 @@ python manage.py dumpdata \
     utils.GitRepoState \
     wordpress.BlogPost \
     sitemaps.SitemapURL \
-    pocketfeed.PocketArticle \
```

Comment:
My only question is whether dropping the model from the proj will mean that a sqlite DB backup can't be loaded into local (or www-sitemap-generator) if it contains any `pocketfeed.PocketArticle` data, because there won't be a table for it any more. That problem would go away if we trigger a fresh DB export without the pocketfeed data anyway. Equally, i don't know if there _is_ any pocketfeed data in the DB these days

Reply:
Steve and I chatted on Slack and this looks fine. No data in the db.